bridges/LuftfahrtBundesAmtBridge.php

```diff
@@ -26,8 +26,13 @@ protected function provideFeedIcon(\DOMXPath $xpath)
     protected function formatItemTimestamp($value)
     {
         $value = trim($value);
-        $dti = DateTimeImmutable::createFromFormat('d.m.Y', $value);
-        $dti = $dti->setTime(0, 0, 0);
+        if (strpos($value, 'Uhr') !== false) {
```

Comment:
protip: `str_contains()`

Reply:
Ah, I thought php 8 was too new, that's why I didn't use it.

Reply:
Hidden away here: https://github.com/RSS-Bridge/rss-bridge/blob/master/lib/php8backports.php